target/linux/ramips/dts/U35WF.dts

```diff
@@ -0,0 +1,113 @@
+/dts-v1/;
+
+/include/ "mt7620n.dtsi"
+
+/ {
+	compatible = "u35wf", "ralink,mt7620n-soc";
+	model = "Kimax U35WF";
+
+	palmbus@10000000 {
```

Comment:
The whole dts file is kind of a mess:

- whitespace issues
- gpio macros not used
- button macros not used
- basically all nodes defined in mt7620.dtsi are redefined instead of referenced

Please have a look at the existing dts files to see how it is done properly